python/lsst/daf/butler/configs/storageClasses.yaml

```diff
@@ -278,3 +278,7 @@ storageClasses:
     converters:
       # Compatible python types.
       lsst.daf.base.PropertySet: lsst.pipe.base.TaskMetadata.from_metadata
+  SpectractorSpectrum:
+    pytype: lsst.atmospec.formatters.Spectrum
+  SpectractorImage:
+    pytype: lsst.atmospec.formatters.Image
```

Comment:
I understand this is a draft PR but these class names seem to imply that you are creating special classes specifically for I/O and they aren't actually Spectractor classes that you are serializing? (putting them in the same hierarchy as the formatter itself).

Reply:
That was temporarily deliberate I think (I was brainstorming this with Jim), but yes, you're right, now that should be changed to the native type, I think. I've pushed the fix.